src/App.vue

```diff
@@ -76,12 +77,65 @@ export default {
 		}
 	},
 
+	computed: {
+		conversations() {
+			return this.$store.getters.conversations
+		},
+		currentToken() {
+			return this.$route.params.token
+		}
+	},
+
+	watch: {
+		currentToken: function() {
+			this.onChangeConversation()
+		}
+	},
+
 	beforeMount() {
 		window.addEventListener('resize', this.onResize)
 		this.onResize()
 	},
 
+	mounted() {
+		setTimeout(this.onChangeConversation, 2500)
```

Comment:
This is a hack, but I failed to get it working without thi, when you page reload inside a conversation

Reply:
can't we add a watcher for the route to change the tab title accordingly?? 
like: 
```
watch: {
$route( ) {
    ...do something
    }
}
```
cc @skjnldsv

Reply:
There u go: https://router.vuejs.org/guide/advanced/navigation-guards.html#global-before-guards

Reply:
Ah, damn, I tried with a watch on `currentToken` and more. but that didnt work.

Thanks for the hint.

Reply:
Okay, so I can use this and it works fine for joining. But not for reloading while you are in a conversation.